mailscanner/database.php

```diff
@@ -41,11 +41,11 @@ private function __construct()
      * @param string $username
      * @param string $password
      * @param string $database
-     * @return mysqli
+     * @return \mysqli
      */
     public static function connect($host = '', $username = '', $password = '', $database = '')
     {
-        if (!self::$link instanceof mysqli) {
+        if (!self::$link instanceof \mysqli) {
```

Comment:
It means that the class is in the root namespace which is the default if you are not setting a namespace in a php file. It is also the namespace where the standard php classes are. If you have a namespace set it is assumed that all classses you are using are in the mentioned namespace and if.you want to use classes in.the root namespace you have to prefix that classes with a `\`. You can also use `use ... as ...` to set a namespace for a single class in the current file